resources/views/docs/info.blade.php

```diff
@@ -259,7 +259,18 @@
 
 For [Authorization Code Grant](#authorization-code-grant) tokens, the Resource Owner is the user authorizing the token.
 
-[Client Credentials Grant](#client-credentials-grant) tokens do not have a Resource Owner (i.e. is a guest user), unless they have been granted the {{ ApidocRouteHelper::scopeBadge('bot') }} scope. The Resource Owner of tokens with the {{ ApidocRouteHelper::scopeBadge('bot') }} scope is the owner of the OAuth Application that was granted the token. Currently, only [Chat Bot]({{ $wikiUrl }})s are allowed to request the {{ ApidocRouteHelper::scopeBadge('bot') }} scope.
+[Client Credentials Grant](#client-credentials-grant) tokens do not have a Resource Owner (i.e. is a guest user), unless they have been granted the {{ ApidocRouteHelper::scopeBadge('bot') }} scope. The Resource Owner of tokens with the {{ ApidocRouteHelper::scopeBadge('bot') }} scope is the owner of the OAuth Application that was granted the token.
+
+
+## Client Credentials Delegation
+
+While Client Credentials Grant tokens normally do not have an associated user by additionally requesting the {{ ApidocRouteHelper::scopeBadge('bot') }} scope, they may be allowed to act on behalf of the owner of the OAuth client (delegation). When using delegation, scopes that support delegation cannot be used together with scopes that do not support delegation.
```

Comment:
```suggestion
While Client Credentials Grant tokens normally do not have an associated user, by additionally requesting the {{ ApidocRouteHelper::scopeBadge('bot') }} scope, they may be allowed to act on behalf of the owner of the OAuth client (delegation). When using delegation, scopes that support delegation cannot be used together with scopes that do not support delegation.
```

I *think* this comma is intended? The meaning is unclear without it.

Reply:
Must have been deleted while rearranging sentences 🤔